src/Castle.Sdk/Castle.Sdk.csproj

```diff
@@ -5,7 +5,7 @@
     <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
     <PackageProjectUrl>https://github.com/castle/castle-dotnet</PackageProjectUrl>
     <RootNamespace>Castle</RootNamespace>
-    <Version>1.2.0</Version>
+    <Version>1.2.1</Version>
```

Comment:
Shouldn't it be 1.3.0 according to semver?

Reply:
Not really adding functionality, but not fixing a bug either I guess. I'll switch